docs/shared/esg_grid.yaml

```diff
@@ -16,7 +16,7 @@ esg_grid:
         pazi: 0.0
```

Comment:
I tidied this file with `yaml-sort` utility, which changed `pazi: 0.0` to `pazi: 0`, though I undid that change. It's probably technically ok, I thought it might be helpful to keep the decimal form for consistency with other values here.